pkg/crypto/aes/aes.go

```diff
@@ -0,0 +1,86 @@
+package aes
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/hex"
+	"bytes"
+	"github.com/Shopify/go-lua"
+	"io"
+)
+
+func Open(l *lua.State) {
+	aesOpen := func(l *lua.State) int {
+		lua.NewLibrary(l, aesLibrary)
+		return 1
+	}
+	lua.Require(l, "goluago/crypto/aes", aesOpen, false)
+	l.Pop(1)
+}
+
+var aesLibrary = []lua.RegistryFunction{
+	{"encryptCBC", encryptCBC},
+	{"decryptCBC", decryptCBC},
+}
+
+func encryptCBC(l *lua.State) int {
+	key, _ := hex.DecodeString(lua.CheckString(l, 1))
```

Comment:
Do we care to report any of the errors along the way? We might want to consider checking and returning 0 for failures along the way. Maybe even add some logging?